data/json/monsters/feral_humans.json

```
@@ -70,9 +71,11 @@
   {
     "id": "mon_feral_human_axe",
     "type": "MONSTER",
+    "description": "Their pupils are dilated and what remains to be seen of the iris and sclera are bloodshot.  This axe-wielding maniac still breathes, but the zombies treat ferals like one of them.",
     "copy-from": "mon_feral_human_pipe",
-    "melee_dice": 3,
+    "melee_dice": 2,
     "melee_dice_sides": 8,
+    "melee_cut": 8,
```

```suggestion
    "melee_dice": 2,
    "melee_dice_sides": 7,
    "melee_cut": 9,
    "extend": { "flags": [ "FILTHY" ] },
```
I calculated out the damage from a fire axe a bit ago. (Filthy is for the blood-stained clothing, as you might guess.)